src/Akka.Cluster.Hosting/SBR/SplitBrainResolverOption.cs

```diff
@@ -84,10 +169,13 @@ public sealed class KeepOldestOption : SplitBrainResolverOption
     /// </summary>
     public sealed class LeaseMajorityOption : SplitBrainResolverOption
     {
+        public override string ConfigPath => SplitBrainResolverSettings.LeaseMajorityName;
+
         /// <summary>
-        /// The absolute HOCON path to the <see cref="Lease"/> implementation HOCON configuration block
+        /// An class instance that extends <see cref="LeaseOptionBase"/>, used to configure the lease provider used in this
+        /// <see cref="LeaseMajority"/> strategy.
         /// </summary>
-        public string LeaseImplementation { get; set; }
+        public LeaseOptionBase LeaseImplementation { get; set; }
```

Comment:
This is an example of direct use of this standard.
Before, the user need to remember that the lease implementation for `Akka.Coordination.KubernetesApi` is `"akka.coordination.lease.kubernetes"`, now they can actually use intellisense and find `KubernetesLeaseOption` (fake name, not implemented yet) instead.